static/src/javascripts/projects/commercial/modules/prebid/slot-config.js

```diff
@@ -11,6 +11,11 @@ import type { PrebidSlot } from 'commercial/modules/prebid/types';
 import { prebidUsMobileSticky } from 'common/modules/experiments/tests/prebid-us-mobile-sticky';
 import { isInVariantSynchronous } from 'common/modules/experiments/ab';
 
+const isMobileStickyPrebidOn = (): boolean =>
+    window.location.hash.indexOf('#mobile-sticky-prebid') !== -1;
+const inMobileStickyPrebidTestOr = (liveClause: boolean): boolean =>
```

Comment:
minor point but is the "Or" at the end of this variable name `inMobileStickyPrebidTestOr` a typo? When reading the name it looks like it should be expressing whether `inMobileStickyPrebidTest...` "or" something else, but that something else is missing.

Reply:
Could you simplify `inMobileStickyPrebidTestOr` to:
```JS
const inMobileStickyPrebidTestOr = (liveClause: boolean): boolean =>
   (config.get('switches.mobileStickyLeaderboard') && isInVariantSynchronous(prebidUsMobileSticky, 'variant')) ||  window.location.hash.indexOf('#mobile-sticky-prebid') !== -1;
```

Doing this means you could remove the `isMobileStickyPrebidOn` function and the `liveClause` parameter passed to `inMobileStickyPrebidTestOr` would no longer be required too.

Reply:
I tried to do it in a more functional way which we can pass a function as the Or condition thus the Or at the end. Similar logic exists in bid-config.js lines 327-329

Reply:
Have now switched back to the suggested simplification